diff --git a/hal/audio_extn/spkr_protection.c b/hal/audio_extn/spkr_protection.c
index e1be59fa..c320829f 100644
--- a/hal/audio_extn/spkr_protection.c
+++ b/hal/audio_extn/spkr_protection.c
@@ -1138,7 +1138,7 @@ static void spkr_calibrate_wait()
     pthread_mutex_unlock(&handle.cal_wait_cond_mutex);
 }
 
-static void* spkr_calibration_thread()
+static void* spkr_calibration_thread(void *context __unused)
 {
     unsigned long sec = 0;
     int t0;
@@ -1915,7 +1915,7 @@ static int get_spkr_prot_v_vali_param(int cal_fd __unused, int *status __unused,
 }
 #endif
 
-static void* spkr_v_vali_thread()
+static void* spkr_v_vali_thread(void *context __unused)
 {
     int ret = 0;
     struct audio_device *adev = handle.adev_handle;
